packages/cli-kit/src/public/node/api/oxygen.ts

```diff
@@ -1,25 +1,17 @@
-import {buildHeaders, debugLogRequest} from './common.js'
-import {graphqlClient} from '../http/graphql.js'
-import {shopifyFetch} from '../http.js'
+import {shopifyFetch} from '../../../http.js'
+import {graphqlRequest} from '../../../private/common/api/graphql.js'
+import {buildHeaders} from '../../../private/common/api/headers.js'
 import {Variables, RequestDocument} from 'graphql-request'
 import FormData from 'form-data'
 import {Response} from 'node-fetch'
 
-export async function request<T>(
+export async function oxygenRequest<T>(
```

Comment:
Similar comment here regarding keeping `graphql-request` as an implementation detail, adding documentation to the function, and moving the non-exported functions to `private/node/api/oxygen`.